app/boards/arm/dao/dao.keymap

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2023 ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+
+#define DEF 0
+#define LWR 1
+#define RSE 2
+#define ADJ 3
+
+&lt {
+    quick_tap_ms = <200>;
+};
+
+&mt {
+    quick_tap_ms = <200>;
+};
+
+/ {
+    keymap {
+        compatible = "zmk,keymap";
+
+        default_layer {
+            // -------------------------------------------------------------------------------------------------
+            // NOTE: For 44 version you should add map to edge buttons
+            // -------------------------------------------------------------------------------------------------
+            // |      | CMD/[  |  Q  |  W  |  E  |  R  |  T  |   |  Y  |  U   |  I  |  O  |  P  | [      |     |
+            //        | CTRL/` |  A  |  S  |  D  |  F  |  G  |   |  H  |  J   |  K  |  L  |  ;  | CTRL/' |
+            //        | ALT/-  |  Z  |  X  |  C  |  V  |  B  |   |  N  |  M   |  ,  |  .  |  /  | ALT/\  |
+            //                | RSE/TAB | SHFT/SPC | LWR/RET |   | LWR/ESC | SHFT/BKSP  | RSE/DEL |
+
+            bindings = <
+  &none    &mt LGUI RBKT   &kp Q   &kp W      &kp E            &kp R      &kp T      &kp Y           &kp U      &kp I    &kp O     &kp P       &kp LBKT   &none
```

Comment:
Normally multiple physical layouts are handled by defining multiple matrix transforms which can be selected by the user in the `chosen` node. You could consider adding a new transform for 44 key layout and modifying the existing (default) one to remove the edge keys, if you think the non-44 key one will be more common.

Reply:
If I understand correctly, the contents of keymap should expose needed matrix. So If I remove extra key in user config I should get a working 42-key keyboard. Am I right?

Reply:
Looks good, but I'd still set the `default_transform` in the `chosen` node in the `dtsi` in case the user writes their keymap from scratch and doesn't put a `chosen` node in it.

Reply:
Moved back.